spec/frisby_json_schema_spec.js

```diff
@@ -159,7 +175,24 @@ describe('Frisby JSONSchema', function() {
     frisby.create(this.description)
       .get('http://example.com/response_x')
       .expectStatus(200)
-      .not().expectJSONSchema('response.data', 'fixtures/json_schema/response1.json')
```

Comment:
This seemed to be a mistake to me. This was supposed to be valid. This is valid now with the use of tv4